plugins/woo-ai/src/product-description/product-description-button-container.tsx

```diff
@@ -161,21 +185,33 @@ export function WriteItForMeButtonContainer() {
 			includedProps.push( name );
 		} );
 
-		return [
+		const instructions = [
 			`Compose an engaging product description for a product named "${ productName.slice(
 				0,
 				MAX_TITLE_LENGTH
-			) }".`,
-			...productPropsInstructions,
-			'Identify the language used in the product name, and craft the description in the same language.',
-			`Ensure the description is concise, containing no more than ${ DESCRIPTION_MAX_LENGTH } words.`,
-			'Structure the content into paragraphs using <p> tags, and use HTML elements like <strong> and <em> for emphasis.',
-			'Only if appropriate, use <ul> and <li> for listing product features.',
-			`Avoid including the properties (${ includedProps.join(
-				', '
-			) }) directly in the description, but utilize them to create an engaging and enticing portrayal of the product.`,
+			) }."`,
+			`Use a 9th grade reading level.`,
+			`Make the description ${ DESCRIPTION_MAX_LENGTH } words or less.`,
+			'Structure the description into paragraphs using standard HTML <p> tags.',
+			'Identify the language used in this product title and use the same language in your response.',
+			'Only if appropriate, use <ul> and <li> tags to list product features.',
+			'When appropriate, use <strong> and <em> tags to emphasize text.',
 			'Do not include a top-level heading at the beginning description.',
-		].join( ' ' );
+		];
+
+		if ( brandingData?.toneOfVoice ) {
+			instructions.push(
+				`Use a ${ brandingData.toneOfVoice } tone of voice.`
```

Comment:
Should we expand a bit on the the tone here in the prompt, such as using something similar to the descriptive text on the settings page? (ie `Passionate and inspiring, engage users with this upbeat tone.` instead of `motivational`)

Reply:
Hmm. This seems to be working decently well with just a single-word value for tone. I think this change would complicate things a bit, since we're not storing "Passionate and inspiring, engage users with this upbeat tone" as a setting value but rather just "motivational", right? The longer explanatory string is essentially just used as a description for each selection. 🤔 

I would like to see us soon try one of the "prompt engineering" tools so we could quickly test lots of variations of the prompt and see objectively and quickly if that would help.

Reply:
We can iterate on this in the future, but I've also found a few minor issues in my testing. 

Sometimes it assumes that the product is in that tone and describes the product as humorous or youthful. For example, I chose a humorous tone for a bookstore, and it sometimes assumed that the books were humorous, too, instead of just generating a description in that tone.

I think expanding a little on the prompt might help. I tweaked it to "Generate the description using the {tone}" and got a little more consistent results.

We could also try moving the instruction up the list near where we say, "Compose an engaging product description..." so that it doesn't confuse it with product properties. I've found that the LLM will put more emphasis on the tone when we mention it in the upper lines of the prompt.

Reply:
I've gone with your suggestion to change the instruction to `Generate the description using a ${ brandingData.toneOfVoice } tone.` 

As a test, I tried having Woo AI write a "humorous" description for a very sad book named Where The Red Fern Grows:

> Rawls’s prose is a symphony of sentimentality, woven with humor and tenderness that will warm your heart. You’ll find yourself laughing out loud and fighting back tears as you witness the extraordinary bond between man and dog.

I think GPT3.5 struggles with a complex task like writing a product description for a serious book using humorous language (that's really difficult!) but GPT4 seems quite a bit better at it in my testing.

A sample from GPT4, which IMO is much better:

> Do you have a heart? Great! Because this book will tug at every single corner of it. Dive into a tale of a young boy, his two hound dogs, and their unyielding bond in the Ozark mountains. Spoiler alert: You might just shed a tear or find yourself googling 'how to care for hound dogs' by the end.

I think we'll really want to start looking at prompt testing + engineering using tools and compare GPT3.5 and 4 outputs against test requirements to check for regressions and/or improvements.
I think the descriptions are quite a bit better with GPT4.